docs/tech-specs/basic-spec.md

```diff
@@ -6,6 +6,280 @@
 | *By* | Imran M Yousuf |
 | *Date* | November 10, 2020 |
 
-## Problem Statement / Goals
+## Problem Statement / Goal
 
-In a service oriented or microservice architecture the necessity to reliable pass messages between systems is essential. We want a reliable message broker without increasing architectural complexity.
+In a service oriented or microservice architecture the necessity to reliably pass messages between systems is essential. We want a reliable message broker without increasing architectural complexity with high throughput and scalability.
```

Comment:
> reliable message broker without increasing architectural complexity with high throughput and scalability

This sentence seems to assume the existing brokers increase architectural complexity. Let's say, the existing brockers have the architectural complexity of `c'` and offers `hs'` throughput and scalability.

How would our implementation stack up with that? (`c` and `hs` for web-broker)? 
1. `c < c' && hs > hs'`
2. `c < c'` && hs == hs'

Reply:
The `hs` component is difficult to predict without proper load stress test; but with `http/2` used and scaling codified I am fairly confident that `hs >= hs'`